demos/embeddings/barbell-graphwave.ipynb

```diff
@@ -0,0 +1,162 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "m1 = 7\n",
+    "m2 = 7\n",
+    "gnx = nx.barbell_graph(m1=m1, m2=m2)\n",
+    "nx.draw(gnx)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### GraphWave embedding calculation\n",
+    "\n",
+    "Now, we're ready to calculate the GraphWave embeddings. We need to specify the points to sample the characteristic function. We need to specify: \n",
+    "\n",
+    "- the wavelet `scales` to the use. This is a graph dependant hyperparameter. Experiment with different values or use the `scales=\"auto\"` option\n",
```

Comment:
```suggestion
    "- a list of wavelet `scales` to use. This is a graph dependent hyperparameter. Experiment with different values.\n",
```
or

```suggestion
    "- a list of wavelet `scales` to use. The best values to use depend on the graph. Experiment with different values.\n",
```

Could we at least give an approximate hint about what's a useful range, so that the user isn't having to start with nothing?

Also, it'd be good to put this sort of thing in the docs, because a user is likely to jump to the API reference for the details (or look at `help(GraphWaveGenerator)` in a REPL/IDE/Jupyter notebook).

Reply:
hm, how about default values for `scales` and an explanation on how changing them will affect the algorithm?

Reply:
Default values is good, and an explanation is even better 👍